ckanext/scheming/validation.py

```diff
@@ -569,14 +569,29 @@ def validator(key, data, errors, context):
     return validator
 
 
+def comma_swap_formatter(arg_value):
+    """
+    Swaps the parts of a string around a single comma.
+    Use to format e.g. "Tanzania, Republic of" as "Republic of Tanzania"
+    """
+    if arg_value.count(',') == 1:
+        parts = arg_value.split(',')
+        stripped_parts = map(lambda x: x.strip(), parts)
+        reversed_parts = reversed(stripped_parts)
+        joined_parts = " ".join(reversed_parts)
+        arg_value = joined_parts
```

Comment:
I think it'd be better to return here `joined_parts` immediately. Better readability and you avoid mutating argument.
```
if ...
   return joined_parts
return arg_value
```

Reply:
We need to return the unaltered arg value otherwise. I could do an if/else and return something different in each branch if you think that is better?

Reply:
Ah sorry. I see what you're saying now. That's fine.

Reply:
But I do think that a proper else branch is more readable and clear personally? 
```
if ...
    return joined_parts
else:
    return arg_value
```